ext/imap/tests/skipif.inc

```diff
@@ -2,13 +2,13 @@
 extension_loaded('imap') or die('skip imap extension not available in this build');
 
 // Change these to make tests run successfully
-$mailbox  = '{localhost/norsh}';
+$mailbox  = '{127.0.0.1:143/debug/norsh}';
```

Comment:
`/debug` intentional?

Reply:
I used it to debug the connection locally and I think it would be rather useful to leave it on